### ISSUE TITLE
New way to set the Explore tab default map

### ISSUE THREAD

**Issue by lisaberrygis** [vannizhang/policymaps-new#17]
It would be nice to have more control over the default map of the Explore tab. Instead of hard-coding a map URL that doesn't change often, we can make it more dynamic to current events by using the Policy maps group. 

I created a new category within Resources called "Default Explore Maps" to test this. 

The site could randomly choose a map from this category. 

I added two for testing:
![image](https://user-images.githubusercontent.com/12399083/111706492-9fa91280-87ff-11eb-9774-bd062acf8bb8.png)

**Comment by vannizhang** [comment]
@lisaberrygis Fixed, please test on DEV: https://livingatlasdev.arcgis.com/policy/browse/

**Comment by lisaberrygis** [comment]
I like that I refreshed a few times and was able to see it swap between the two different maps I put in there. Is there a way to force the scale as well as the extent? I noticed that when I tested with different screen sizes I could see some huge differences in the default scale. 

Ex:
![image](https://user-images.githubusercontent.com/12399083/112062049-99c37200-8b1c-11eb-9621-d9a3ec157e85.png)
![image](https://user-images.githubusercontent.com/12399083/112062060-9cbe6280-8b1c-11eb-82ca-be4be780e778.png)
 
Is there a way to always force the map to appear at the 18 million scale? The new map viewer has an option to preserve scale, so it might be something new to the 4x API. 

In the meantime, I will show this to the team on Thursday to see what they think.
